### PR TITLE
common-SLES12-2016.10: sles12 deps for scorep

### PR DESCRIPTION
**Pull request by jgphpc**
## Logs
```  
X=/apps/common/UES/sandbox/jgp/easybuild.dom/gpu/easybuild/software

$X/Score-P/3.0-CrayCCE-2016.11/easybuild/easybuild-Score-P-3.0-20161125.170105.log
$X/Score-P/3.0-CrayGNU-2016.11/easybuild/easybuild-Score-P-3.0-20161125.165525.log
$X/Score-P/3.0-CrayPGI-2016.11/easybuild/easybuild-Score-P-3.0-20161125.170009.log
$X/Score-P/3.0-CrayIntel-2016.11/easybuild/easybuild-Score-P-3.0-20161125.170155.log
```

### CODE REVIEW

File: jenkins-builds/common-SLES12-2016.10 [hunk 1]
```diff
@@ -14,4 +14,13 @@ likwid-4.1.2.eb
 HDFView-2.13.0.eb
 ctags-5.8.eb
 lynx-2.8.8rel.2.eb
-tmux-2.3.eb
\ No newline at end of file
+tmux-2.3.eb
+# --- scorep
```
Comment (jgphpc) [inline jenkins-builds/common-SLES12-2016.10]: Is it because of jenkins ?

File: jenkins-builds/common-SLES12-2016.10 [hunk 2]
```diff
@@ -14,4 +14,13 @@ likwid-4.1.2.eb
...
+zlib-1.2.8-scorep.eb
```
Comment (jgphpc) [inline jenkins-builds/common-SLES12-2016.10]: score-p depends on compiler&mpich, there is no such thing as a generic version.

Comment (gppezzi) [inline jenkins-builds/common-SLES12-2016.10]: then what is the point of having generic dependencies on a separate software stack? just to save build time?

**Comment by gppezzi** [comment]
```
perettig@daint101:~> ls -ld /apps/common/UES/sandbox/jgp/
drwx--S--- 127 piccinal csstaff 8192 Nov 25 12:51 /apps/common/UES/sandbox/jgp/
```

File: jenkins-builds/common-SLES12-2016.10 (hunk 2 above)
Comment (jgphpc) [inline jenkins-builds/common-SLES12-2016.10]: > we need to build 'common' and then the system-specific one
This does not make sense: the common stack is compatible with SLES12 based systems and all compilers hence i do not get why you would need to rebuild it for a specific system...

Comment (jgphpc) [inline jenkins-builds/common-SLES12-2016.10]: > we need to change all jenkins projects.

why ? just need modulepath to point to /apps/common/UES/SLES12/easybuild/modules/all

**Comment by gppezzi** [comment]
Tests passed on dom and daint by doing:
```
module rm PrgEnv-cray/6.0.3
module use /opt/cray/pe/craype/2.5.8/modulefiles
```

Looks good, thanks @jgphpc !

**Pull request merged**